README.md

```diff
@@ -23,7 +23,7 @@ Markdown is a lightweight option for formatting text. See this [cheat sheet](htt
 |Chase|   |
 |Chris|   |
 |Colin|   |
-|Danny|   |
+|Danny| @dannylopez  |
```

Comment:
Hi @dlopez101 ,

Your GitHub Handle is not `@dannylopez` but `@dlopez101`. Can you please make that change?

Reply:
Good afternoon Dr. Link,

I resolved the issue, thanks for bringing it to my attention.